### DISCUSSION TITLE
Changing Licenses

### DISCUSSION THREAD

**Discussion by Kfeavel** [panix-os-dev-team/Panix#182] [General]
Alright, time to talk about software licenses. Originally I chose MIT because I like the absolute freedom it provides, but as Panix starts to develop, I've changed my perspective and want to protect Panix to a certain degree.

Ideally, I would like to require forks or distributions to provide acknowledgment of their original source, but it would also be nice if there was an expectation of helping further Panix.

The GNU LGPLv3 license stands out because it requires attribution and source code distribution for the original source, but if it is part of a larger work, the other pieces can have a different license.

I know that 3 or 4 Clause BSD is also something I've looked into in the past.

**Comment by micahswitzer** [comment]
I'm totally okay with MIT, but if it must change I'm much more in favor of BSD. I'd avoid GPL if at all possible.

**Reply by Kfeavel** [reply]
I'm usually not a fan of GPL as well, but I'm curious to hear your reasons for Panix specifically.

**Reply by Kfeavel** [reply]
If we were to go with BSD, I'd prefer 4-clause BSD, which would look can be found [here](https://spdx.org/licenses/BSD-4-Clause.html).
